frontend/src/environments/environment.ts

```diff
@@ -4,7 +4,7 @@
 
 export const environment = {
   production: false,
-  host: "http://localhost:8080"
+  host: 'http://localhost:8080'
```

Comment:
Probably we should consider using docker image urls

Reply:
We use docker urls in production, this is just for local dev, as the frontend is not behind an nginx proxy, but runs on a webpack dev server. We could probably look around in the webpack config, but I'm not sure how to do this currently.